lab_3_generate_by_ngrams/main.py

```diff
@@ -159,6 +217,22 @@ def _postprocess_decoded_text(self, decoded_corpus: tuple[str, ...]) -> Optional
 
         In case of corrupt input arguments, None is returned
         """
+        if not isinstance(decoded_corpus, tuple) or len(decoded_corpus) < 1:
+            return None
+        string_text = ''
+        string_text = ''.join([string_text, decoded_corpus[0].upper()])
```

Comment:
Consider using a method that is made specifically for capitalizing the first letter (try googling it)

it would make your code easier to read

Reply:
thank you :)
i think you told me about capitalize. I tried to use it there)